README.md

```diff
@@ -9,194 +9,192 @@
-// join the red channel and broadcast data to subscribers
-await fdc3.joinChannel('red')
-fdc3.broadcast(instrument)
+> As an open standard, FDC3 can be implemented on any platform and in any language.
+> All that is required is a "Desktop Agent" that implements the FDC3 standard, which is responsible for co-ordinating application interactions.
+> FDC3 is successfully running on Web and Native platforms in financial institutions around the world.
 
-// set up a listener for incoming data
-const listener = fdc3.addContextListener('fdc3.contact', contact => { })
-```
+### 🔌  End the Integration Nightmare
 
-### Installation
+> By providing support for FDC3, vendors and financial organizations alike can avoid the bilateral or trilateral integration projects that plague desktop app roll-out, cause vendor lock-in and result in a slow pace of change on the Financial Services desktop.
 
-To access the APIs in your application, simply install the FDC3 npm package:
+### 👐 Open Standards Promote Innovation
 
-```sh
-# npm
-npm install @finos/fdc3
+> FDC3 is developed collaboratively by a [consortium of industry participants](https://fdc3.finos.org/community#type-all) including banks, agent vendors, app developers and FinTech firms.  By design, FDC3 is open to extension.  We have an active community working on growing and improving the standard with new data and intents.
 
-# yarn
-yarn add @finos/fdc3
+# How Does It Work?
 
-# pnpm
-pnpm install @finos/fdc3
-```
+FDC3 includes a standardized API for a Desktop agent, an OpenAPI App Directory, standardized verbs to invoke actions between applications (called "intents") and standardized formats for data passed between applications (called "context data").
 
-Note that the web application still needs to run in the context of an FDC3 desktop agent to work. For more details, please see the [relevant documentation](https://fdc3.finos.org/docs/supported-platforms).
+Hence, the standard currently consists of four parts:
 
-### Usage
+1. [API Part](https://fdc3.finos.org/docs/api/spec)
+2. [App Directory Part](https://fdc3.finos.org/docs/app-directory/overview)
+3. [Intents part](https://fdc3.finos.org/docs/intents/spec)
+4. [Context Data Part](https://fdc3.finos.org/docs/context/spec)
 
-FDC3 can be accessed either via the global window object:
+The specifications are informed by agreed [business use cases](https://fdc3.finos.org/docs/use-cases/overview),
+and implemented and used by leading [financial industry participants](https://fdc3.finos.org/community#type-all).
 
-```ts
-if (window.fdc3) {
-  const channel = await window.fdc3.getOrCreateChannel('myChannel')
-  channel.broadcast({ ... })
-}
-```
+See https://fdc3.finos.org for more information, including on [Compliance] and the [FDC3 charter], as well as a comprehensive [API Reference]. You can also take the free [FDC3 Training](https://www.edx.org/course/fdc3-interoperability-for-the-financial-desktop) for an introduction to FDC3's core concepts and usage.
```

Comment:
@robmoffat should we also link the [FDC3 znglist](https://resources.finos.org/category/projects-sigs/fdc3/?projects-sigs=fdc3) where marketing consolidate educational material for FDC3 users?

Reply:
I've linked it in the community page, and referenced the community page here.  There's way more information on the community page than in the znglist atm.